explorer/README.md

```diff
@@ -138,9 +135,12 @@ Please refer to
 
 ## Trace Program Execution
 
-When tracing is turned on (using the `--trace_file=...` option), `explorer`
-prints the state of the program and each step that is performed during
-execution.
+When tracing is turned on (using the `--trace_file=...` option or `.verbose`
+file extension), `explorer` prints the state of the program and each step that
```

Comment:
```suggestion
When tracing is turned on (using the `--trace_file=...` option or `.verbose`
target), `explorer` prints the state of the program and each step that
```

`.verbose` is a target name, rather than a file. I think avoiding calling it a file extension would help avoid confusion about whether there's a matching file.